src/main/scala/nl.knaw.dans.easy.stage/dataset/EMD.scala

```diff
@@ -52,6 +58,33 @@ object EMD extends DebugEnhancedLogging {
     }
   }
 
+  def addAgreementFields(emd: EasyMetadata)(implicit s: Settings): Unit = {
+    val agreementPath = s.bagitDir.toPath.resolve(Paths.get("metadata/agreements.xml"))
+    if (Files.exists(agreementPath)) {
+      val agreementsXml = XML.loadFile(agreementPath.toFile)
+      if (BooleanUtils.toBoolean((agreementsXml \\ "depositAgreementAccepted").text)) {
+        emd.getEmdRights.setAcceptedLicense(true)
+      }
+      if (BooleanUtils.toBoolean((agreementsXml \\ "containsPrivacySensitiveData").text)) {
+        emd.getEmdOther.getEasRemarks.add(new BasicRemark("containsPrivacySensitiveData"))
```

Comment:
We need to get agreement on this `containsPrivacySensitiveData` keyword. Is this already a supported keyword? Do we already have another one? @lindareijnhoudt thought?

Reply:
`“Message for the Datamanager: according to the depositor <signerID> this dataset DOES (NOT) contain Privacy Sensitive data.”`